config/localmodelnodes/manager.yaml

```
@@ -65,6 +66,8 @@ spec:
       volumes:
         - name: models
           hostPath:
-            path: /models
+#            path: /models
+#            changed to work with Openshift permissions, re-evaluate as a part of https://issues.redhat.com/browse/RHOAIENG-33247
+            path: /var/lib/models
```

perhaps, try to "namespace" it: 
```suggestion
            path: /var/lib/odh/models
```